collectcontexts.py

```diff
@@ -5,22 +5,22 @@
 
 from tf.fabric import Fabric
 
-from hebrewreader import DATADIR, FEATURES, load_data
+from hebrewreader import DATADIR, FEATURES, GRK_FEATURES, SYR_FEATURES, load_data
 from minitf import gather_context
 
 VERSE_NODES = dict()
 
-def gather_chapter(api, book, chap):
+def gather_chapter(api, book, chap, lang):
```

Comment:
Please make `lang` the second argument, so that the order matches the one in the `VERSE_NODES` hierarchy.